css/main.css

```diff
@@ -0,0 +1,698 @@
+
+body  {
+  width: 1300px;
+  margin: 0 auto;
+}
+
+/* CSS Off Head */
+
+header h1 {
+  font-family: 'Lato', sans-serif;
+  font-weight: 100;
+  font-size: 300px;
+  width: 480px;
+  float: left;
+  margin-top: 0px;
+  padding: 30px 0px 300px 120px;
+  color: white;
+  line-height: .85;
+  position: relative;
+  z-index: 5;
+  letter-spacing: 1px;
+}
+
+.css_spacer {
+  letter-spacing: 27px;
+}
+
+p {
+  font-family: 'Merriweather', serif;
```

Comment:
It's best to apply this to the `body` element and not specifically to a `p` tag. This will only lead to duplication of this rule set.